### DISCUSSION TITLE
On a Quarto blog, can one set up a listing by author?

### DISCUSSION THREAD

**Discussion by maelle** [quarto-dev/quarto-cli#6243] [Q&A] [answered]
### Description

Is there a way to set up listings by author similar to the listings by categories https://quarto.org/docs/websites/website-listings.html#categories? I was also hoping to link author names to their post listings. From what I see in the docs, the taxonomy support in Quarto is (understandably!) less complete than with, say, Hugo?

Sorry in advance if I missed something. :pray:

**Comment by dragonstyle** [comment, marked as answer]
Right now there isn't an automatic way to do this. Starting with Quarto 1.3, you can create listings that filter based upon a property (like authors), so you could create different listings that show only specific authors (though this isn't quite what you're looking for, I think):

This example would be a listing that shows any posts with the author `Harlow` or `Tristan`:

```yaml
listing:
  contents: posts
  type: grid
  include:
    author: "{Harlow,Tristan}*"
```

This doesn't really support the idea of linking from the author name either (e.g. in the post metadata)... I definitely agree that this would be a very nice feature for listings to support. I'll create an enhancement request for this to tracking tying to get to this soon!

**Reply by maelle** [reply]
Thank you so much! I'll follow the corresponding issue then. :smile_cat:

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/6246

**Reply by maelle** [reply]
So choosing an answer doesn't "close" the discussion? Could I as a question asker have closed it? Asking for the next time I ask a question :wink:

**Reply by mcanouil** [reply]
Yes, you could.

"We" are closing only if there is a "better" place to keep the discussion going such as an issue opened from a discussion like in this case.
I explicitly write I closed and linked to the issue, because only on desktop view of the website you can see the linked issue (on the right).
<img width="276" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/cb1d396d-0b5d-431c-8026-47208f0f8e5b">

Also, by default on GitHub you don't search in closed items which help concentrate the discussion again in the chosen "better" place.
<img width="678" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/e740595b-9cbd-42c2-84c0-c427bbdd351c">

**Reply by maelle** [reply]
Thank you, good to know :pray:
